src/main/resources/hudson/plugins/copyartifact/monitor/LegacyJobConfigMigrationMonitor/manage.jelly

```diff
@@ -95,15 +95,15 @@ THE SOFTWARE.
                                         <tr>
                                             <td class="first-column">
                                                 <j:if test="${!isValid}">
-                                                    <img class="first-column-icon" src="${imagesURL}/16x16/warning.png" title="${%InvalidWarning.title}" />
+                                                    <l:icon style="padding: 0 0 0 4px" class="icon-warning icon-sm" title="${%InvalidWarning.title}" />
```

Comment:
Using class is preferred to using `style`.
I will see whether appropriate alternate class is available (I think I can do that this weekend).

Reply:
.first-column-icon is the one defined here: https://github.com/jenkinsci/copyartifact-plugin/blob/copyartifact-1.46.3/src/main/resources/hudson/plugins/copyartifact/monitor/LegacyJobConfigMigrationMonitor/resources.css#L30

I'll update the css to applicable to here.